specs/card_spec.rb

```diff
@@ -0,0 +1,46 @@
+
+require_relative 'spec_helper'
+
+describe Card do
+  describe "You can create a Card instance" do
+
+    it "Can be created" do
+      [:hearts, :spades, :clubs, :diamonds].each do |suit|
+        (1..13).each do |value|
+          card = Card.new(value, suit)
+          card.must_be_instance_of Card
+        end
+      end
+    end
+
+    it "Raises an ArgumentError for invalid parameters" do
+      proc {Card.new 0, :diamonds}.must_raise ArgumentError
```

Comment:
This is a bit confusing without parens so could we update to be `{ Card.new(0, :diamonds) }`?